CRM/Odoosync/Sync/Request/Auth.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/**
+ * Login in Odoo API and gets Odoo user id
+ */
+class CRM_Odoosync_Sync_Request_Auth {
+
+  /**
+   * Instance for singleton
+   *
+   * @var object
+   */
+  protected static $instance;
+
+  /**
+   * Setting for Odoo sync
+   *
+   * @var array
+   */
+  protected $setting;
+
+  /**
+   * Odoo userId
+   *
+   * @var int
+   */
+  public $odooUserId = FALSE;
+
+  /**
+   * Odoo login url
+   *
+   * @var string
+   */
+  const ODOO_LOGIN_URL = '/xmlrpc/2/common';
```

Comment:
this is not a login URL, it is the API endpoint, so a better name for this constant would be : 

ODOO_API_ENDPOINT